packages/terra-base/tests/nightwatch/ArrayBase.jsx

```diff
@@ -5,14 +5,10 @@ const ArrayBase = () => (
   <div>
     <Base>
       <div id="one">One</div>
```

Comment:
only one child is needed to verify this works

Reply:
This test was to show multiple sibling children can be passed to Base, we'll want to keep the multiple sibling children.

Reply:
Technically this is a better test for jest IMO